js/modules/modules.go

```diff
@@ -34,6 +37,38 @@ func Register(name string, mod interface{}) {
 		panic(fmt.Sprintf("module already registered: %s", name))
 	}
 	modules[name] = mod
+	getPackageVersion(mod)
+}
+
+func getPackageVersion(mod interface{}) {
```

Comment:
```suggestion
func getModuleVersion(mod interface{}) {
```

I think this is slightly clearer, as we are getting the Go _module_ version after all (and the map is called `moduleVersions`).

Reply:
good catch.

btw, If a package has two modules, it just has one package in map.
how about display the module name? e.g.

```
 extensions: github.com/dgzlopes/k6-extension-notifications k6/x/ext1 v1.0.0
                     github.com/dgzlopes/k6-extension-notifications k6/x/ext2 v1.0.0
```